### PR TITLE
Conda skeleton pypi conversion for Pyobjc

### PR DESCRIPTION
**Pull request by kastman**

### CODE REVIEW

File: recipes/pyobjc-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/pyobjc-core/meta.yaml]: Do we need `setuptools` at run time? Could we try dropping it?

File: recipes/pyobjc-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - PyObjCTools
+    - objc
+
+about:
+  home: http://pyobjc.sourceforge.net/
+  license: MIT
```
Comment (jakirkham) [inline recipes/pyobjc-core/meta.yaml]: Is there a license file that we can include from the source in the built package?

File: recipes/pyobjc-core/meta.yaml (hunk 1 above)
Comment (kastman) [inline recipes/pyobjc-core/meta.yaml]: Commit #00cc7b4 didn't include it and I think I added it after failing Ci, but I don't remember the details. This compiles fine without it locally, so I'm going to try sending it through CI again?

File: recipes/pyobjc-core/meta.yaml (hunk 2 above)
Comment (kastman) [inline recipes/pyobjc-core/meta.yaml]: Yep - license file added.

**Comment by kastman** [comment]
Setuptools is needed at runtime - it's imported in one of the modules. Here's the stack trace when it's not included at runtime:

```
source /Users/travis/miniconda/bin/activate /Users/travis/miniconda/conda-bld/pyobjc-core_1485134476439/_t_env
+/Users/travis/miniconda/conda-bld/pyobjc-core_1485134476439/_t_env/bin/python -s /Users/travis/miniconda/conda-bld/pyobjc-core_1485134476439/test_tmp/run_test.py
===== testing package: pyobjc-core-3.2.1-py27_0 =====
import: 'PyObjCTools'
import: 'objc'
Traceback (most recent call last):
  File "/Users/travis/miniconda/conda-bld/pyobjc-core_1485134476439/test_tmp/run_test.py", line 30, in <module>
    import objc
  File "/Users/travis/miniconda/conda-bld/pyobjc-core_1485134476439/_t_env/lib/python2.7/site-packages/objc/__init__.py", line 32, in <module>
    from objc._bridgesupport import *
  File "/Users/travis/miniconda/conda-bld/pyobjc-core_1485134476439/_t_env/lib/python2.7/site-packages/objc/_bridgesupport.py", line 13, in <module>
    import pkg_resources
ImportError: No module named pkg_resources
TESTS FAILED: pyobjc-core-3.2.1-py27_0
```

File: recipes/pyobjc-core/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{% set version = "3.2.1" %}
+{% set md5 = "df752f952825dd68958c28a9b07967b6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/7e/03/303a5c7f7c3d3af811eba44b32ef957e570be4d5c5b656c0b44ece6191e0/{{ name }}-{{version}}.tar.gz
```
Comment (jakirkham) [inline recipes/pyobjc-core/meta.yaml]: Please replace with the URL in [`example`]( https://github.com/conda-forge/staged-recipes/blob/a0222417ed30935a383f5773ff21d23322247cdd/recipes/example/meta.yaml#L18 ).

File: recipes/pyobjc-core/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{% set name = "pyobjc-core" %}
+{% set version = "3.2.1" %}
+{% set md5 = "df752f952825dd68958c28a9b07967b6" %}
```
Comment (jakirkham) [inline recipes/pyobjc-core/meta.yaml]: Please use `sha256` from [pypi.org]( https://pypi.org/ ).

File: recipes/pyobjc-core/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{version}}.tar.gz
```
Comment (jakirkham) [inline recipes/pyobjc-core/meta.yaml]: nit: Please add spaces around `version`.

File: recipes/pyobjc-core/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not osx]
```
Comment (jakirkham) [inline recipes/pyobjc-core/meta.yaml]: Was there a reason to skip macOS? Sorry I may have forgotten.

**Comment by jakirkham** [comment]
Thanks @kastman. Sorry for the long turnaround.

**Pull request merged**